### PR TITLE
Update tqdm to 4.67.*

### PR DESCRIPTION
**Pull request by yangdanny97**
closes https://github.com/python/typeshed/pull/12968

### CODE REVIEW

**Comment by yangdanny97** [comment]
Added mypy suppression because the source just has `**kwargs` but it overrides a function without `kwargs`.

File: stubs/tqdm/tqdm/contrib/discord.pyi [hunk 1]
```diff
@@ -79,8 +81,9 @@ class tqdm_discord(tqdm_auto[_T]):
         gui: bool = ...,
         **kwargs,
     ) -> None: ...
+    # type: ignore[override]
     def display(
-        self, msg: str | None = ..., pos: int | None = ..., close: bool = ..., bar_style=..., check_delay: bool = ...
+        self, *, msg: str | None = ..., pos: int | None = ..., close: bool = ..., bar_style=..., check_delay: bool = ...
```
Comment (brianschubert) [inline stubs/tqdm/tqdm/contrib/discord.pyi:86]: mypy failure is due to these parameters being keyword-only here but positional-or-keyword in the parent:

https://github.com/python/typeshed/blob/90842a4520785e0f6ad5d00fccac6a57ca6c6ad3/stubs/tqdm/tqdm/std.pyi#L224

Comment (yangdanny97) [reply stubs/tqdm/tqdm/contrib/discord.pyi:86]: This seems like an issue in the source library, the implementation for this function is keyword-only: https://github.com/tqdm/tqdm/blob/master/tqdm/contrib/discord.py#L126

Comment (JelleZijlstra) [reply stubs/tqdm/tqdm/contrib/discord.pyi:86]: Then we'll need to add a type ignore here with a comment.

File: stubs/tqdm/tqdm/contrib/discord.pyi [hunk 2]
```diff
@@ -8,10 +9,15 @@ from .utils_worker import MonoWorker
 __all__ = ["DiscordIO", "tqdm_discord", "tdrange", "tqdm", "trange"]
 
 class DiscordIO(MonoWorker):
+    API: str = "https://discord.com/api/v10"
+    UA: str = ...
     text: Incomplete
     message: Incomplete
```
Comment (srittau) [inline stubs/tqdm/tqdm/contrib/discord.pyi:15]: The `token`, `channel_id`, and `session` attibutes were also added in this release: https://github.com/tqdm/tqdm/compare/v4.66.6...v4.67.0#diff-f6ef645a2466cf282cfa55fb82f064281e15b1437c1dde109b172c7195e6ed4fR34

**Review by srittau — CHANGES_REQUESTED** [review]

File: stubs/tqdm/tqdm/contrib/discord.pyi (hunk 2 above)
Comment (yangdanny97) [reply stubs/tqdm/tqdm/contrib/discord.pyi]: fixed

**Review by srittau — APPROVED** [review]
Thanks!

**Pull request merged**